plasma_cash/client/db/db_interface.py

```diff
@@ -0,0 +1,12 @@
+import abc
+
+
+class DbInterface(abc.ABC):
```

Comment:
I think we also need something like get_data_for_block(block_num), which is used when client want to submit exit or challenge